### PR TITLE
DP-24907: Adjust image display on decision tree content type and add to backstop

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-24907

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: backstop/all.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   {"label": "CuratedListPeopleContact", "url": "/lists/qag-curated-list2-people"},
   {"label": "DecisionMandate", "url": "/mandate/qag-decisionmandate", "screens": ["desktop", "mobile"]},
   {"label": "DecisionOrder", "url": "/order/qag-decisionorder", "screens": ["desktop", "mobile"]},
+  {"label": "DecisionTree", "url": "/decision-tree/qa-decision-tree", "screens": ["desktop", "mobile"]},
```
Comment (clairesunstudio) [inline backstop/all.json:24]:
```suggestion
  {"label": "DecisionTree", "url": "/decision-tree/qa-decision-tree", "screens": ["desktop", "mobile"]},
    {"label": "DecisionTreeBranch", "url": "/decision-tree/qa-decision-tree#s=125376", "screens": ["desktop", "mobile"]},
  
```

Comment (clairesunstudio) [reply backstop/all.json:24]: please also add this page to backstop

**Review by ygannett — COMMENTED** [review]
@arthurbaghdas The change looks good, but please look into the PHP unit error and Minghua's comment for the backstop reference setup.  The ticket mentions to set up 2 references for binder as she pointed out.  Thank you!

**Comment by arthurbaghdas** [comment]
Thank you @ygannett , I will merge this once we get Unit tests issue fixed.

**Pull request merged**